Terminal.Gui/Input/Key.cs

```diff
@@ -1,5 +1,7 @@
+using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
+using static System.Runtime.CompilerServices.RuntimeHelpers;
```

Comment:
This namespace is really needed?

Reply:
I haven't looked at this yet but, following onto @BDisp's question, I'd add this suggestion:

`using static` should be avoided most of the time, because it makes it unclear that method calls dependent on it actually came from the static class referenced in the `using static`. It also makes naming collisions more perilous.

About the only place I've ever really thought it was a good idea is when dealing with expression trees, since everything is a static method in there and the code is half as long with the using than without it.

Reply:
Fixed.